meilisearch-http/src/task.rs

```diff
@@ -90,7 +90,7 @@ impl From<Task> for TaskResponse {
             index_uid,
             content,
             events,
-        } = task;
+        } = dbg!(task);
```

Comment:
```suggestion
        } = task;
```

Reply:
yes sorry this must be merged after #1902, since it is partially based on it